virt-v2v/cold/constants.go

```diff
@@ -0,0 +1,182 @@
+package main
+
+const CheckConnectivityBash = `#!/bin/bash
```

Comment:
We could use the `go:embed` which would allow us to add scripts into separate files and load them using the embedded filesystem. 

It is also supported by the bezel 
https://github.com/bazelbuild/rules_go/blob/master/docs/go/core/embedding.md

Reply:
Nice, thank you , I will use the embed